Resources/apiresources.robot

```diff
@@ -923,3 +923,13 @@ checkAlarmStatusForAllAHUsInGroup
         #log to console  ${json_dictionary["data"]}
         log to console  ==================${alarm_type} Alarm Cleared for ahu ${ahu_list}[${i}]====================
     END
+
+    #Created by Greeshma on 28th Dec 2021
+overrideAllAHUsWithBOPValueON
```

Comment:
overrideAllAHUsWithBOPValueOFF can be parameterized to pass the value 1or 0

Reply:
added comment